src/containers/Login/Login.tsx

```diff
@@ -39,48 +41,98 @@ export function Login() {
       .catch((err) => console.error(err));
   };
 
-  if (didLoginSucceed) {
-    return <Redirect to="/admin" />;
+  const handlePharmacistSubmit = () => {
+    setIsPharmacist(true);
+    setName("PharmacyUser");
+    setPassword(pharmacyPassword);
+    post({ name, password })
+      .then((user) => {
+        if (user.key) {
+          userService.setUser(user);
+          setState({ user });
+          setDidLoginSucceed(true);
+        } else {
+          setIsInvalidCredentials(true);
+        }
+      })
+      .catch((err) => console.error(err));
+  };
+
+  if (!isPharmacist && didLoginSucceed) {
```

Comment:
Don't rely on just a single boolean for redirection - `isPharmacist` is initialized with false, so it may be possible for staff to get redirected with this code block. Add a `isStaff` boolean.